uwblueprint/focus-on-nature
# Form Management: Form Template- Question Card & View Only Page Layout

## Notion ticket link

[Form Management: Form Template- Question Card & View Only Page Layout](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=0c868083f92e49b797e0beda597f859a&pm=c)



## Implementation description
* 



## Steps to test
1. go to global forms page
2. You should see what's on the template (check against db)
3. You should be able to add a section and see the changes reflected on the page and the db



## What should reviewers focus on?
* common components (we can abstract more but it would take longer to refactor older code imo -> eg: WaiverSectionCard specifically)


## Checklist
